src/rastervision/common/utils.py

```diff
@@ -15,6 +15,8 @@
     s3_results_path, results_path, s3_datasets_path, datasets_path,
     s3_weights_path, weights_path, s3_bucket)
 
+def eprint(*args, **kwargs):
```

Comment:
eprint is useful in that it prints out to cloudwatch logs; perhaps we should leave this until we have a better logging mechanism?